Python/bytecodes.c

```diff
@@ -1418,6 +1419,7 @@ dummy_func(
             _PyStackRef temp = retval;
             DEAD(retval);
             SAVE_STACK();
+            DTRACE_FUNCTION_RETURN();
```

Comment:
You've covered `return` and `yield`, but you'll need a probe point for unwinding as well.